Documentation/CodeSnippets/Extbase/Controllers/ForwardAction.rst.txt

```diff
@@ -0,0 +1,47 @@
+.. Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets 
+.. Extracted from EXT:blog_example/Classes/Controller/PostController.php
+
+.. code-block:: php
+   :caption: EXT:blog_example/Classes/Controller/PostController.php
+
+   use FriendsOfTYPO3\BlogExample\Domain\Model\Blog;
+   use Psr\Http\Message\ResponseInterface;
+   use TYPO3\CMS\Core\Pagination\SimplePagination;
+   use TYPO3\CMS\Extbase\Http\ForwardResponse;
+   use TYPO3\CMS\Extbase\Pagination\QueryResultPaginator;
+   
+   class PostController extends \FriendsOfTYPO3\BlogExample\Controller\AbstractController
+   {
+       /**
+        * Displays a list of posts. If $tag is set only posts matching this tag are shown
+        */
+       public function indexAction(
+           Blog $blog = null,
+           string $tag = '',
+           int $currentPage = 1
+       ): ResponseInterface {
+           if ($blog == null) {
+               return (new ForwardResponse('index'))
```

Comment:
Is it possible to highlight the relevant lines (24-27) by the code snippet tool? The example does also other things, so this would be nice as additional hint where to look.

Reply:
done